models/Author.js

```diff
@@ -1,54 +1,66 @@
-'use strict';
+import mongoose from 'mongoose';
+import mockgoose from 'mockgoose';
 
-var mongoose = require('mongoose');
-var mongodb = require('common/makeCredential')('mongodb');
+if (process.env.NODE_ENV === 'test') {
+  mockgoose(mongoose);
+}
 
-var db = mongoose.createConnection(mongodb);
+const mongodbCredential = process.env.KINDLIZED_MONGODB;
+const db = mongoose.createConnection(mongodbCredential);
 
-var AuthorSchema = new mongoose.Schema({
+const AuthorSchema = new mongoose.Schema({
   name: {
     type: String,
     index: {
-      unique: true
-    }
+      unique: true,
+    },
   },
   lastModified: Date,
-  pageId: Number
+  pageId: Number,
 });
 
-var SequenceSchema = new mongoose.Schema({
+const SequenceSchema = new mongoose.Schema({
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).